awx/ui_next/src/screens/Inventory/InventorySources/InventorySourceListItem.test.jsx

```diff
@@ -0,0 +1,91 @@
+import React from 'react';
+import { act } from 'react-dom/test-utils';
+import { mountWithContexts } from '@testUtils/enzymeHelpers';
+import InventorySourceListItem from './InventorySourceListItem';
+
+const source = {
+  id: 1,
+  name: 'Foo',
+  source: 'Source Bar',
+  summary_fields: { user_capabilities: { start: true, edit: true } },
+};
+describe('<InventorySourceListItem />', () => {
+  afterEach(() => {
+    jest.clearAllMocks();
+  });
+  test('should mount properly', async () => {
+    let wrapper;
+    const onSelect = jest.fn();
+    await act(async () => {
```

Comment:
This component doesn't perform any effects or state updates, so I don't think we need to mount it with `await act(async () `